### ISSUE TITLE
Multiple BLE connections alternating so buffer doesnt get overflood

### ISSUE THREAD

**Issue by DaanP-2056102** [adabru/BleWinrtDll#45]
Hello I managed to connect 3 BLE devices and everything works super fine, thanks for the DLL.

But for my application (thesis). I have to read data simultaneously meaning that I am reading 0.5 seconds the data from Sensor1, then 1 second reading my sensor 2, and then sensor 3. The buffer data always gets messed up and my code is reading the wrong sensor for each data field. Is it possible to switch between connected BLE devices and then make a scheduler to indicate when every sensor is allowed to read or is this simply not possible with your DLL?  Thanks in advance

**Comment by adabru** [comment]
Yeah that's probably confusing because "ReadCharacteristic" is not implemented in the dll although it is a ble functionality. If you need it you can checkout https://github.com/adabru/BleWinrtDll/issues/6#issuecomment-1480375448 .
